packages/taxdat/R/plot_DroppedData_table.R

```diff
@@ -33,7 +33,7 @@ plot_DroppedData_table <- function(config, cache, cholera_directory, aesthetic =
   used_obs_stats <- obs_stats[order(obs_stats$year), ] %>% 
     dplyr::filter(!grepl('imputed',u_OCs))
   #get all obs_stats
-  get_stan_input(name="stan_input",config=config, cache=cache, cholera_directory=cholera_directory)
+  get_sf_cases(name="sf_cases",config=config, cache=cache, cholera_directory=cholera_directory)

```

Comment:
I don't understand how this function worked before if we had not loaded sf_cases into cache[["sf_cases"]]...

Reply:
I think it was loaded outside the package in the Rmarkdown file (here: https://github.com/HopkinsIDD/cholera-mapping-pipeline/blob/dev/Analysis/output/country_data_report.Rmd#L131)